src/applications/vaos/new-appointment/components/ConfirmationPage/ConfirmationDirectScheduleInfoV2.jsx

```diff
@@ -61,7 +61,7 @@ export default function ConfirmationDirectScheduleInfoV2({
           <div className="vads-u-display--inline"> {typeOfCare}</div>
         </>
       )}
-      <div className="vads-u-display--flex vads-u-flex-direction--column small-screen:vads-u-flex-direction--row">
+      <div className="vads-u-display--flex vads-u-flex-direction--column small-screen:vads-u-flex-direction--row vads-u-margin-top--neg2">
         <div className="vads-u-flex--1 vads-u-margin-top--2 vads-u-margin-right--1 vaos-u-word-break--break-word">
```

Comment:
Instead of adding a className with negative margin value in the flex container, you can remove the className `vads-u-margin-top--2` inside the nested div's. The other nested div is on line 79 
```suggestion
      <div className="vads-u-display--flex vads-u-flex-direction--column small-screen:vads-u-flex-direction--row ">
        <div className="vads-u-flex--1  vads-u-margin-right--1 vaos-u-word-break--break-word">
```

Reply:
OMG Jennie nice catch....that is embarrassing.....:)